standalone/pruntime/scripts/check-calls.py

```diff
@@ -0,0 +1,99 @@
+#!/usr/bin/env python3
+import re
+import sys
+import r2pipe
+
+
+def error(msg):
+    print(msg, file=sys.stderr)
+
+
+class Checker:
+    SYSCALL_WHITELIST = [
+        re.compile(r)
+        for r in [
+            # We have no way to patch the syscalls used in std::sync::Once unless to patch the std itself.
+            # The `Once` uses thread::park() which futher calls futex.
+            # Since we let the futex syscall return immediately, the logic in `Once` is equivalent to using a spinlock.
+            r"sym.std::sync::once::Once::call_inner::h[0-9a-f]+",
+            r"sym._std::sync::once::WaiterQueue_as_core::ops::drop::Drop_::drop::h[0-9a-f]+",
+        ]
+    ]
+
+    def __init__(self, filename):
+        self.p = r2pipe.open(filename)
+        self.p.cmd("aa;aae;")  # run analysis
+
+    def check_syscalls(self):
+        p = self.p
+        supported_syscalls = {
+            213,  # SYS_epoll_create
+            283,  # SYS_timerfd_create
+            286,  # SYS_timerfd_settime
+            287,  # SYS_timerfd_gettime
+            291,  # SYS_epoll_create1
```

Comment:
epool_create1 is allowed but not epool_wait, etc?

Reply:
The others are not called by the `syscall` function. There are `fn epoll_wait`,etc., to serve them.

Reply:
OK